app/screens/AppNavigator/screens/Balances/BalancesNavigator.tsx

```diff
@@ -93,6 +101,23 @@ export function BalancesNavigator (): JSX.Element {
           headerBackTitleVisible: false
         }}
       />
+      <BalanceStack.Screen
+        name='SendConfirmationScreen'
+        component={SendConfirmationScreen}
+        options={{
+          headerBackTitleVisible: false,
+          headerTitle: () => {
+            return (
+              <View style={tailwind('flex-row items-center')}>
```

Comment:
You can use <HeaderTitle text={translate('screens/SendConfirmationScreen', 'Confirm Send')} />, here